integration/oneup_custom/Tiltfile

```diff
@@ -0,0 +1,9 @@
+# -*- mode: Python -*-
+
+custom_build(
+  'gcr.io/windmill-test-containers/integration/oneup-custom',
+  'docker build -t $TAG .',
```

Comment:
out of curiosity, how does this image get pushed?

Reply:
(a better test might be to use a non-docker image builder...)

Reply:
So the builder returns a ref that the image_build_and_deployer pushes, just like the regular docker_build. :)